src/pipecat/services/cartesia.py

```diff
@@ -73,6 +73,8 @@ def __init__(
             encoding: str = "pcm_s16le",
             sample_rate: int = 16000,
             language: str = "en",
+            speed: str = None,
+            emotion: list = None,
```

Comment:
I'm thinking that instead of adding more arguments we could do like `FalImageGenService` or `GladiaSTTService`:

```
class CartesiaTTSService(TTSService):
    class InputParams(BaseModel):
        speed: Optional[str] = None,
        emotion: Optional[List[str]] = None
```

Reply:
In that case, wouldn’t it be better to include the other options like encoding, sample_rate, and language in the InputParams as well?

Reply:
Yes, I just didn't want to ask you to do more work :sweat_smile:

Reply:
No problem, I’ve made a patch based on your advice.

The 'voice_id' parameter is required, so I’ve left it as the default parameter.

Please review and let me know if everything looks good.

Reply:
Hmm… maybe we need something like TTSEmotionUpdateFrame and TTSSpeedUpdateFrame? 🤔

Reply:
Not needed for now.